test/lintConfigTest.js

```diff
@@ -0,0 +1,109 @@
+/*
+*
+* This file is a straight copy of lint config test for the purpose of testing
+* the editor config file
+*
+*/
+
+var a, b; // Lint is set to "intialization"; only error assign a or b
+var a, b, c = 0; // This should error
+
+let d,
+    e;
+
+let f,
+    g = 0;
+
+//this is a bad comment, (no space, no cap)
+// this is not a bad (despite no caps) because it's a consecutive comment
```

Comment:
@phuchle i fixed the expected behaviour. Look at ln 21-26 for the error that was expected previously